### ISSUE TITLE
replace colorbox with bootstrap modal?

### ISSUE THREAD

**Issue by RobertKorteland** [SocialSchools/socialschools-cms#85]
@vinitkumar I was wondering: wouldn't it be better to replace colorbox with bootstrap modal? Since we're already using bootstrap and its more consistent with other modal views? Or is there a good reason to use colorbox?

**Comment by vinitkumar** [comment]
@RobertKorteland Yes, I think the colorbox is there for legacy reasons since it has already been there. If and when we want to move to bootstrap modal, I would suggest let's upgrade to bootstrap3 too.
